src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -405,6 +405,28 @@ p, li { white-space: pre-wrap; }
          </layout>
         </widget>
        </item>
+       <item>
+        <widget class="QGroupBox" name="autoFrameTrackGroupBox">
+         <property name="title">
+          <string>Auto FrameTrack</string>
+         </property>
+         <layout class="QVBoxLayout" name="verticalLayout_9">
+          <item>
+           <widget class="QCheckBox" name="autoFrameTrackCheckBox">
+            <property name="toolTip">
+             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A frame track will be added when enabling this option at the start of every session.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+            </property>
+            <property name="accessibleName">
+             <string>AutoFrameTrackCheckBox</string>
+            </property>
+            <property name="text">
+             <string>Automatically add the default frame track</string>
```

Comment:
Using "the" in "the default frame track" sounds a bit strange to me, because there is not a canonically and uniquely defined frame track (there are choices in the function used, and whether to base it on start or end of the timeslice, etc.)

Suggestion:
"Automatically add a render frame track by default."

Reply:
Added the suggestion. Also the missing ⓘ that indicates that there is a tooltip.